static/settings.js

```diff
@@ -78,13 +78,7 @@ define(function (require) {
 
     function setupSettings(root, settings, onChange) {
         settings = settings || {};
-        if (settings.delayAfterChange === undefined)
-            settings.delayAfterChange = 750;
-        if (settings.colouriseAsm === undefined)
-            settings.colouriseAsm = true;
-        if (settings.hoverShowSource === undefined)
-            settings.hoverShowSource = true;
-
+      
```

Comment:
Were those lines not necessary or am I missing their replacement somewhere else on the code? I got a merge conflict so I guess they were not necessary